collectd/files/df.conf

```diff
@@ -10,9 +10,9 @@
 LoadPlugin df
 
 <Plugin df>
-{%- if collectd_settings.plugins.df.Device is defined and collectd_settings.plugins.df.Device %}
-      Device "{{ collectd_settings.plugins.df.Device }}"
-{%- endif %}
+{%- for device in collectd_settings.plugins.df.Devices %}
+      Device "{{ device }}"
+{%- endfor %}
```

Comment:
Indeed, I forgot to mention.
On the one hand I could just add ``Devices`` variable without removing ``Device``, but that would look confusing for new users.
What about returning the ``Device`` variable and marking it as obsolete?